packages/neon/neon/lib/src/utils/request_manager.dart

```diff
@@ -34,26 +38,31 @@ class RequestManager {
 
   Cache? _cache;
 
-  Future<void> wrapNextcloud<T, R>(
+  Future<void> wrapNextcloud<T, R, H>(
```

Comment:
```suggestion
  Future<void> wrapNextcloud<T, B, H>(
```
Body and Headers again